packages/css/tests/shorthand-parser.test.ts

```diff
@@ -151,7 +151,7 @@ describe("Font shorthand", () => {
         "fontFamily": "Georgia,serif",
         "fontSize": "12px",
         "fontStyle": "italic",
-        "fontWeight": 700,
+        "fontWeight": "bold",
```

Comment:
A relatively harmless but unintended side effect of this change is that the `bold` keyword no longer transforms into `700`. However, it’s worth noting that in the previous version, a weight like `bolder` would have inaccurately become `700`.